pontoon/base/tests/test_utils.py

```diff
@@ -19,10 +20,27 @@
 
 
 def test_util_glob_to_regex():
-    assert glob_to_regex('*') == '^.*$'
-    assert glob_to_regex('/foo*') == '^\\/foo.*$'
-    assert glob_to_regex('*foo') == '^.*foo$'
-    assert glob_to_regex('*foo*') == '^.*foo.*$'
+    assert glob_to_regex('*') == '^([^/]*)$'
+    assert glob_to_regex('*foo') == '^([^/]*)foo$'
+    assert glob_to_regex('*foo*') == '^([^/]*)foo([^/]*)$'
+
+
+@pytest.mark.skipif(
+    sys.version_info[0] > 2,
+    reason="re.escape escapes non-alphanum characters differently between Python 2 and Python 3."
+)
+def test_util_glob_to_regex_python2():
+    assert glob_to_regex('{ variable }/foo*') == '^\/foo([^/]*)$'  # noqa: W605
```

Comment:
I think you can fix the `W605` by using `r'^\/foo([^/]*)$'`

Reply:
Nice catch, thanks! I'll push fixes tomorrow.